app/scripts/app.js

```diff
@@ -47,6 +48,16 @@ blocList.controller('Landing.controller', ['$scope', '$firebaseArray', function(
     };
 }]);
 
-blocList.controller('Second.controller', ['$scope', function($scope) {
-    $scope.subText = "Barnie and Friends!";
+blocList.controller('History.controller', ['$scope', '$firebaseArray', function($scope, $firebaseArray) {
+    var url = 'https://bloc-list.firebaseio.com/todos';
+    var fireRef = new Firebase(url);
+
+    $scope.todos = $firebaseArray(fireRef);
+
+    $scope.hideActive = function () {
+        var expiryDate = this.todo.expiryDate;
```

Comment:
I think this could be more clear, and in the template, if `$scope.hideActive` takes a todo arg. Then, in your template you could use `hideActive(todo)`.